.extlib/simplesamlphp/modules/saml/src/Auth/Source/SP.php

```diff
@@ -306,25 +306,9 @@ public function getHostedMetadata(): array
      */
     public function getIdPMetadata(string $entityId): Configuration
     {
-        // auth_saml2 modification.
```

Comment:
can we leave the auth_saml2 string in place? makes it easy to grep for modifications in the lib